ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-log_path = ./ansible.log
+log_path = ./logs/ansible.log
```

Comment:
Should we add the logs folder in the .gitignore list? Since this will keep adding to that

Reply:
yeah we currently have *.log in gitignore, we should add the logs directory too @smrafey